### ISSUE TITLE
3.0.0: exception: 'in <string>' requires string as left operand, not type

### ISSUE THREAD

**Issue by hugovk** [sloria/sphinx-issues#125]
With 3.0.0 I get this:

```console
$ make doc
/Library/Developer/CommandLineTools/usr/bin/make -C docs html
sphinx-build -b html -W --keep-going -d _build/doctrees   . _build/html
Running Sphinx v4.3.2
making output directory... done

Extension error (sphinx.config):
Handler <function check_confval_types at 0x102e716c0> for event 'config-inited' threw an exception (exception: 'in <string>' requires string as left operand, not type)
make[1]: *** [html] Error 2
make: *** [doc] Error 2
```

2.0.0 is fine.

`git bisect` points to 2e019f877028ee5d3dfc35165a11ac686dee2150 from PR https://github.com/sloria/sphinx-issues/pull/120, ping @CarliJoy:

```
2e019f877028ee5d3dfc35165a11ac686dee2150 is the first bad commit
commit 2e019f877028ee5d3dfc35165a11ac686dee2150
Date:   Tue Jan 11 02:47:22 2022 +0100

    Allow customization (#120)

    * allow custom uris for everything

    * use sphinx config defaults instead of customs

    * add tests and documentation for customization features

    * fix typo

    * fix travis

    * fix travis old release

    * okay travis really only wants -dev

    officially they support Python 3.10 non dev but it seems they suck
    https://travis-ci.community/t/add-python-3-10/12220

    * add change log for setting

    * Copy edits; update changelog

    Co-authored-by: Steven Loria <sloria1@gmail.com>

 README.rst            |  87 ++++++++++---
 sphinx_issues.py      | 349 +++++++++++++++++++++++++++++++++++++-------------
 test_sphinx_issues.py | 137 +++++++++++++++++++-
 3 files changed, 463 insertions(+), 110 deletions(-)
```

**Comment by CarliJoy** [comment]
Could you provide the Makefile? And the conf.py?

**Comment by hugovk** [comment]
Yes, it's these:

* https://github.com/python-pillow/Pillow/blob/main/Makefile
* https://github.com/python-pillow/Pillow/blob/main/docs/conf.py

Something like this to repro:

```
git clone https://github.com/python-pillow/Pillow
cd Pillow
python -m pip install -r requirements.txt
make doc
```

I'm using Python 3.10 on macOS.

I tried replacing `issues_github_path = "python-pillow/Pillow"` with `issues_default_group_project = "python-pillow/Pillow"` in conf.py but got the same result, also with both defined.

**Comment by CarliJoy** [comment]
I have a look....

**Comment by CarliJoy** [comment]
Found the issue -> the type is documented wrong in sphinx, it has to be a list of types and not a string...

I fixed it already but I am also writing a complete integration unit test, so this does not happen again.

**Comment by hugovk** [comment]
Thank you!

**Comment by CarliJoy** [comment]
Done fixed. Only  @sloria has to merge it.

**Comment by sloria** [comment]
Fix released in 3.0.1

**Comment by hugovk** [comment]
Confirmed 3.0.1 working, thank you both for the quick fix and release!
